renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -1452,6 +1452,14 @@ void WrappedOpenGL::ActivateContext(GLWindowingData winData)
       m_Internal = m_Real;
 
       glEmulate::EmulateRequiredExtensions(&m_Real, &m_Internal);
+
+      // Initialize VBOs used in case we copy from client memory.
+      glGenBuffers(ARRAY_COUNT(m_ClientMemoryVBOs), m_ClientMemoryVBOs);
+      for(size_t i = 0; i < ARRAY_COUNT(m_ClientMemoryVBOs); i++)
+      {
+        glBindBuffer(eGL_ARRAY_BUFFER, m_ClientMemoryVBOs[i]);
+        glBufferData(eGL_ARRAY_BUFFER, 64, NULL, eGL_DYNAMIC_DRAW);
+      }
```

Comment:
This will create one set of buffers per context and overwrite the previous IDs which I think could cause problems.

I agree having one set per context is the right thing to do as otherwise it's a headache trying to track when to create them in case people create a context and destroy it and create another and so on. However I think that means we'll need to move this array into the per-context data.

It should work pretty much the same with the rest of the code I _think_ - the implicit current context will mean when we bind and when we check for these buffers we look up the current context's arrays, we don't have to worry about cross-over.